proxysql/tests/conftest.py

```diff
@@ -0,0 +1,98 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+from copy import deepcopy
+
+import pymysql
+import pytest
+
+from datadog_checks.dev import TempDir, docker_run, get_docker_hostname, get_here
+from datadog_checks.dev.conditions import CheckDockerLogs, WaitFor
+
+DOCKER_HOST = get_docker_hostname()
```

Comment:
nit: we usually put this elsewhere so there are no import from conftest, but we only use these here atm so no worries